app/konnect/getting-started/configure-mesh.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Configure a Global Control Plane with Kong Mesh
+content_type: tutorial
+beta: true
+---
+
+Using Mesh Manager, you can create {{site.mesh_product_name}} global control planes to manage your {{site.konnect_saas}} mesh services. 
+
+A global control plane is a managed central component that stores and distributes (to local zones) all of the configuration and policies for your meshes and services. The global control planes are responsible for validating and accepting connections from local zone control planes and distributing the appropriate configuration down to each local zone as required. They also serve as the target for all `kumactl` CLI operations when manipulating resources and configuration within the mesh deployment.
+
+![mesh global control plane](/assets/images/diagrams/gslides/kuma_multizone.svg)
```

Comment:
Do we need a better image that makes explicit what runs in Konnect versus what users still run?

Reply:
@lahabana I think a different image could be good. If you or someone on the team can mock one up, I can help with words and send it to marketing to make a final image we can add here.

Reply:
Tracking this for GA update on https://konghq.atlassian.net/browse/DOCU-3165